benchmark/models/xception/xception.py

```diff
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import timm
+from timm.data import resolve_data_config
+from timm.data.transforms_factory import create_transform
+from datasets import load_dataset
+from torch.utils.data import DataLoader
+from ...common import (
+    DummyCVDataset,
+    ImageNetDataset,
+    benchmark_model,
+    torch_df_from_str,
+)
+import torch
+from pybuda._C.backend_api import BackendDevice
```

Comment:
Move this import statement after line 26, under the `if device == "tt"` statement, because if the user is running on a GPU-only system they wont have pybuda installed and this will prevent an import error.